lib/SILOptimizer/Mandatory/DiagnoseUnreachable.cpp

```diff
@@ -297,6 +294,81 @@ static bool constantFoldEnumTerminator(SILBasicBlock &BB,
   return true;
 }
 
+static bool constantFoldEnumAddrTerminator(
```

Comment:
This is just a copy of constantFoldEnumTerminator without my changes above to the object version.